### PR TITLE
Prepare mentoring for GSoC

### PR DESCRIPTION
**Pull request by Ana06**
Add OpenBuildService, openQA and Hackweek tool to mentoring. Put at the top of the list of project the projects that will participate in GSoC.

image

### CODE REVIEW

File: _posts/2018-01-02-project-open-qa.markdown [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+layout: default
+img: openqa.png
+title: openQA
```
Comment (Ana06) [inline _posts/2018-01-02-project-open-qa.markdown]: @foursixnine can you double check that the information about openQA is correct? 🙏

Comment (foursixnine) [reply _posts/2018-01-02-project-open-qa.markdown]: Yes it is correct

**Comment by ChrisBr** [comment]
Why is it necessary to rename all the projects?
Also I'm not sure if it is worthwhile to keep the empty projects, they're anyway in git...

**Comment by ChrisBr** [comment]
You would also need to update the GSoC page because it contains the information from last year:
http://101.opensuse.org/gsoc/

**Comment by ChrisBr** [comment]
I'm not sure if we need a dedicated section for all projects (e.g. trollolo does not even have a website) and just bundle these small projects with only one project in the Other section.

**Comment by Ana06** [comment]
@ChrisBr updated! I didn't have time to update the http://101.opensuse.org/gsoc/ section, but I think we can merge this, so that the projects already look updated and I'll send next week another PR. 😉

File: _posts/2018-01-02-project-open-qa.markdown [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+type: project
+technologies: [Perl]
+website: http://open.qa/
+github: https://github.com/os-autoinst/openQA
```
Comment (foursixnine) [inline _posts/2018-01-02-project-open-qa.markdown]: Actually you can leave it just at the organizational level if you want

Comment (Ana06) [reply _posts/2018-01-02-project-open-qa.markdown]: Can I change or should I change it? 😉

Comment (foursixnine) [reply _posts/2018-01-02-project-open-qa.markdown]: It's completely up to you :) Project is openQA but the org is os-autoinst 📄

**Pull request merged**